README.md

```diff
@@ -1,33 +1,36 @@
-# balena-configuration-dsl
+# Configuration DSL to JSON Schema
 
-This library serves as an entry point to the reconfiguration ecosystem.
+## What is it ?
 
-For the main usecase the API surface needs to be similar to (work in progress):
+This is is a Rust (edition 2018) library that is intended to be used as a translation mechanism between the [configuration DSL](https://github.com/balena-io/balena/blob/832f5551127dd8e1e82fa082bea97fc4db81c3ce/specs/configuration-dsl.md) and the `JSON + UI Schema` world.  
```

Comment:
1) I'd remove this link, because when we finish, spec will be public, we will copy & paste it to the repo (not now). For now, it's useless, because it points to a private repo.

2) There's nothing like JSON + UI Schema world. It's JSON Schema and UI Object.

Think about people coming outside of balena. This is public repo.

Reply:
I think it would nice to add links to these two things:

* http://json-schema.org/
* https://github.com/mozilla-services/react-jsonschema-form#the-uischema-object